citation/paper.md

```diff
@@ -0,0 +1,146 @@
+The provided API also allows to quickly compare, sort, and inspect query versus reference spectra using either the included similarity scores or added custom measures.
+
+Matchms is freely accessible either as conda package (https://anaconda.org/nlesc/matchms), or in form of source-code on GitHub (https://github.com/matchms/matchms).
+All main functions are covered by tests and continuous integration to offer reliable functionality.
+We explicitly value future contributions from a mass spectrometry interested community and hope that matchms can serve as a reliable and accessible entry point for handling complex mass spectrometry datasets using Python. 
+
+
+# Example workflow
+A typical workflow with matchms looks as indicated in \autoref{fig:flowchart}, or as described in the following code example.
+```python
+from matchms.importing import load_from_mgf
+from matchms.filtering import default_filters
+from matchms.filtering import normalize_intensities
+from matchms import calculate_scores
+from matchms.similarity import CosineGreedy
+
+# Read spectrums from a MGF formatted file
+file = load_from_mgf("all_your_spectrums.mgf")
+
+# Apply filters to clean and enhance each spectrum
+spectrums = []
+for spectrum in file:
+    spectrum = default_filters(spectrum)
+    spectrum = normalize_intensities(spectrum)
+    spectrums.append(spectrum)
+
+# Calculate Cosine similarity scores between all spectrums
+scores = calculate_scores(references=spectrums,
+                          queries=spectrums,
+                          similarity_function=CosineGreedy())
+
+# Print the calculated scores for each spectrum pair
+for score in scores:
+    (reference, query, score, n_matching) = score
+    # Ignore scores between same spectrum and
+    # pairs which have less than 20 peaks in common
+    if reference != query and n_matching >= 20:
+        print(f"Reference scan id: {reference.metadata['scans']}")
+        print(f"Query scan id: {query.metadata['scans']}")
+        print(f"Score: {score:.4f}")
+        print(f"Number of matching peaks: {n_matching}")
+        print("----------------------------")
+```
+
+![Matchms provided a range of filter functions to process spectrum peaks and metadata. Filters can easily be stacked and combined to build a desired pipeline. The API also makes it easy to extend customer pipelines by adding own filter functions. \label{fig:filtering}](filtering_sketch.png)
+
+# Processing spectrum peaks and plotting
+Matchms provides numerous filters to process mass spectra peaks. Below a simple example to remove low intensity peaks from a spectrum (\autoref{fig:peak_filtering}).
+```python
+from matchms.filtering import require_minimum_number_of_peaks
+from matchms.filtering import select_by_mz
+from matchms.filtering import select_by_relative_intensity
+
+def post_process(s):
```

Comment:
What happened before the post processing? It should be called `process` instead of `post_process` or maybe just `filter`.

Reply:
Changed it to ``process_peaks`` because it combines filters that only affect peaks.

Reply:
Thanks